### PR TITLE
chore: protect individual certificate endpoint with token

### PR DESCRIPTION
**Pull request by Nil20**

### CODE REVIEW

File: src/data-seeding/certificates/handler.ts [hunk 1]
```diff
@@ -13,36 +13,8 @@ import { Request, ResponseToolkit } from '@hapi/hapi'
 import { readFileSync } from 'fs'
 
 export async function certificateHandler(request: Request, h: ResponseToolkit) {
```
Comment (rikukissa) [inline src/data-seeding/certificates/handler.ts:15]: Let's move this endpoint outside `data-seeding` directory altogether

**Comment by rikukissa** [comment]
@Nil20 Can you also provide a PR in country config repository?

**Comment by Nil20** [comment]
> @Nil20 Can you also provide a PR in country config repository?

Yep done

**Comment by Nil20** [comment]
Country config PR: https://github.com/opencrvs/opencrvs-countryconfig/pull/250

**Pull request closed without merging**